ddtrace/llmobs/_constants.py

```diff
@@ -35,15 +35,19 @@
 OUTPUT_TOKENS_METRIC_KEY = "output_tokens"
 TOTAL_TOKENS_METRIC_KEY = "total_tokens"
 
-EVP_PROXY_AGENT_BASE_PATH = "evp_proxy/v2"
-EVP_PROXY_AGENT_ENDPOINT = "{}/api/v2/llmobs".format(EVP_PROXY_AGENT_BASE_PATH)
+EVP_PROXY_AGENT_BASE_PATH = "/evp_proxy/v2"
+EVP_PROXY_SPAN_ENDPOINT = "{}/api/v2/llmobs".format(EVP_PROXY_AGENT_BASE_PATH)
+EVP_PROXY_EVAL_ENDPOINT = "{}/api/intake/llm-obs/v2/eval-metric".format(EVP_PROXY_AGENT_BASE_PATH)
 EVP_SUBDOMAIN_HEADER_NAME = "X-Datadog-EVP-Subdomain"
-EVP_SUBDOMAIN_HEADER_VALUE = "llmobs-intake"
+EVP_SPAN_SUBDOMAIN_HEADER_VALUE = "llmobs-intake"
+EVP_EVAL_SUBDOMAIN_HEADER_VALUE = "api"
 EVP_PAYLOAD_SIZE_LIMIT = 5 << 20  # 5MB (actual limit is 5.1MB)
 EVP_EVENT_SIZE_LIMIT = (1 << 20) - 1024  # 999KB (actual limit is 1MB)
 
-AGENTLESS_BASE_URL = "https://llmobs-intake"
-AGENTLESS_ENDPOINT = "api/v2/llmobs"
+AGENTLESS_SPAN_BASE_URL = "https://llmobs-intake"
+AGENTLESS_EVAL_BASE_URL = "https://api"
+AGENTLESS_SPAN_ENDPOINT = "/api/v2/llmobs"
```

Comment:
Switching from the `HTTPWriter` to `BaseLLMObsWriter` (using `httplib.HTTPSConnection()` directly vs the `HTTPWriter` specific connection code) requires adding the leading `/` in the endpoint